src/api/spec/models/authenticator_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+require 'authenticator'
+require 'gssapi'
+
+RSpec.describe Authenticator do
```

Comment:
Just noticed this but you've created the spec here: `spec/models/authenticator_spec.rb` but `Authenticator` is not a model its in the lib directory so shouldn't the spec go in `spec/lib/authenticator_spec.rb`?